nucypher/config/storages.py

```diff
@@ -296,6 +296,11 @@ class LocalFileBasedNodeStorage(NodeStorage):
     _name = 'local'
     __METADATA_FILENAME_TEMPLATE = '{}.node'
 
+    __identity_method = lambda self, x: x
```

Comment:
Incredible feat of serialization.

Reply:
I know, right? In a subsequent rebasing commit, I changed the concept from "serializer" to "encoder", as that's what it really is, a bytes encoder. In this case, encoding bytes to hex strings it's not very useful and it's simply wasting resources.

Reply:
To be clear, hex encoding is the current `main` behavior, as that's what the superclass `NodeStorage` defines.

Reply:
✅